### PR TITLE
Add flex recipe for SuluFormBundle

### PR DESCRIPTION
**Pull request by luca-rath**
| Q             | A
| ------------- | ---
| License       | MIT

Link to package: https://github.com/sulu/SuluFormBundle

### CODE REVIEW

**Comment by luca-rath** [comment]
ping @alexander-schranz

File: sulu/form-bundle/2.0/config/packages/sulu_form.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+sulu_form:
+    mail:
+        from: "%env(SULU_ADMIN_EMAIL)%"
+        to: "%env(SULU_ADMIN_EMAIL)%"
+        sender: "%env(SULU_ADMIN_EMAIL)%"
+
+framework:
+    esi: true
```
Comment (alexander-schranz) [inline sulu/form-bundle/2.0/config/packages/sulu_form.yaml:8]: Not happy with this but currently needed. Did create an issue for it on our side: https://github.com/sulu/SuluFormBundle/issues/265

**Review by alexander-schranz — APPROVED** [review]

File: sulu/form-bundle/2.0/config/packages/sulu_form.yaml (hunk 1 above)
Comment (luca-rath) [reply sulu/form-bundle/2.0/config/packages/sulu_form.yaml:8]: 👍

**Comment by alexander-schranz** [comment]
@nicolas-grekas the CI is currently failing because in sulu the project need to define which `phpcr/phpcr-implementation` it will use. Not sure if there is any way to detect that. Maybe by checking the "composer.json" of the bundle/library has `jackalope/jackalope-doctrine-dbal` or `jackalope/jackalope-jackrabbit` in its "require-dev" like [here](https://github.com/sulu/SuluFormBundle/blob/fe518a1929099a986a2d19413639b77bcd58b5f2/composer.json#L40)  and then run e.g.:

```bash
composer require jackalope/jackalope-doctrine-dbal --no-update
```

Before running composer install script.

**Comment by Nyholm** [comment]
It is really tricky to find virtual packages. No worries about the CI failures.

File: sulu/form-bundle/2.0/config/packages/sulu_form.yaml (hunk 1 above)
Comment (Nyholm) [reply sulu/form-bundle/2.0/config/packages/sulu_form.yaml:8]: I agree. Im not happy with this either. 

Thank you for the issue. We should make sure these are removed when https://github.com/sulu/SuluFormBundle/issues/265 is fixed.

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**